aiida_common_workflows/workflows/relax_and_bands.py

```diff
@@ -0,0 +1,324 @@
+# -*- coding: utf-8 -*-
+"""
+Workflow that runs a relaxation and subsequently calculates bands.
+It can use any code plugin implementing the common relax workflow and the
+common bands workflow.
+It also allows the automatic use of `seekpath` in order to get the high
+symmetries path for bands.
+"""
+import inspect
+
+from aiida import orm
+from aiida.common import AttributeDict, exceptions
+from aiida.engine import ToContext, WorkChain, calcfunction, if_
+from aiida.orm.nodes.data.base import to_aiida_type
+from aiida.plugins import WorkflowFactory
+
+from aiida_common_workflows.workflows.bands.generator import CommonBandsInputGenerator
+from aiida_common_workflows.workflows.bands.workchain import CommonBandsWorkChain
+from aiida_common_workflows.workflows.relax.generator import CommonRelaxInputGenerator, RelaxType
+from aiida_common_workflows.workflows.relax.workchain import CommonRelaxWorkChain
+
+
+def deserialize(inputs):
```

Comment:
This is now only used to convert the `seekpath_parameters` inputs to Python base types in order to wrap them in a `Dict`. I would say that this is overkill, because they only contain `Float`s and a `Bool`. So you only need one conditional. I would just make the conversion explicit without a function.

Better still, why wrap the nodes in a `Dict`? It would be better to have the `calcfunction` simply accept the same input nodes as the `seekpath_parameters` namespace. This way provenance will be direct.